cirq-google/cirq_google/api/v2/result.proto

```diff
@@ -55,14 +58,15 @@ message QubitMeasurementResult {
   // Which qubit was measured.
   Qubit qubit = 1;
 
-  // These are the results of a measurement on a qubit. Measurement results are
-  // repetitions number of bits, where the repetitions are define in the
-  // sweep result message.
+  // These are the results of a measurement on a qubit. The number of bits
+  // measured is equal to repetitions * instances, where repetitions is defined
+  // in the SweepResult message, and instances is defined in the MeasurementResult
+  // message.
   //
   // The bytes in this field are constructed as follows:
   //
   // 1. The results of the measurements produce a list of bits ordered by
-  //    the round of repetition.
+  //    the round of repetition and instance within a round.
```

Comment:
I didn't notice any tests that confirm this behavior. Can you add them or point me to them? Thanks!

Reply:
Added tests.

Reply:
I see tests that you can serialize and deserialize with the expected result, but not that the serialization follows the contract listed in this proto.